tests/gds_kernel_latency.c

```diff
@@ -218,7 +221,7 @@ static struct pingpong_context *pp_init_ctx(struct ibv_device *ib_dev, int size,
         ctx->txbufexp_lkey      = NULL;
         ctx->txbufexp_addr      = NULL;
 
-        size_t alloc_size = max_batch_len * align_to(size + 40, page_size);
+        size_t alloc_size = max_batch_len * ctx->buf_align;
```

Comment:
since you are using memalign/posix_memalign below, why do you need buf_align in the first place?
it should not be needed, as those allocators already provide buffers with the right size.

Reply:
This is from previous version of the code. I'll remove it

Reply:
Should I keep using size + 40?

Reply:
+40 is the padding required for the UD protocol. 
You can removed it if we make sure UD cannot be selected in this test.